js/researches/french/firstWordExceptions.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Returns an array with exceptions for the sentence beginning researcher.
+ * @returns {Array} The array filled with exceptions.
+ */
+module.exports = function() {
+	return [ "le", "la", "les", "un", "une", "deux", "trois", "quatre", "cinq", "six", "sept", "huit", "neuf",
```

Comment:
I would categorize these words and add a comment before all groups description what the words are so it is clear why they are exceptions.